### PR TITLE
Point in Time Endpoints

### PR DESCRIPTION
**Pull request by philkra**
added the request & response for:
* `open_point_in_time`
* `close_point_in_time`

### CODE REVIEW

**Review by sethmlarson — DISMISSED** [review]
LGTM

File: output/typescript/types.ts [hunk 1]
```diff
@@ -7855,6 +7866,14 @@ export interface OpenJobResponse extends ResponseBase {
   opened: boolean
 }
 
+export interface OpenPointInTimeRequest extends RequestBase {
+  index: IndexName
```
Comment (stevejgordon) [inline output/typescript/types.ts:7870]: Should these be here? I believe the request for this is empty and the index and keep_alive come from the path and query?

File: output/typescript/types.ts [hunk 2]
```diff
@@ -7855,6 +7866,14 @@ export interface OpenJobResponse extends ResponseBase {
...
+  keep_alive?: Time
+}
+
+export interface OpenPointInTimeResponse extends ResponseBase {
+}
```
Comment (stevejgordon) [inline output/typescript/types.ts:7875]: Should this not include the returned ID?

File: output/typescript/types.ts (hunk 1 above)
Comment (philkra) [reply output/typescript/types.ts:7870]: `keep_alive` is a query param and `index` is in the url path according to the docs: https://www.elastic.co/guide/en/elasticsearch/reference/master/point-in-time-api.html#point-in-time-api-example

Comment (stevejgordon) [reply output/typescript/types.ts:7870]: Sorry, didn't realise I was looking at the output!

File: output/typescript/types.ts (hunk 2 above)
Comment (philkra) [reply output/typescript/types.ts:7876]: done with https://github.com/elastic/elastic-client-generator/pull/215/commits/c0a2c666de5c642c73b33602abf3d896bc598a65 - great catch, thanks!

Comment (sethmlarson) [reply output/typescript/types.ts:7876]: Shouldn't this have been caught via validator? Or are there no tests for PIT APIs?

**Review by sethmlarson — APPROVED** [review]

File: output/typescript/types.ts (hunk 2 above)
Comment (philkra) [reply output/typescript/types.ts:7876]: I had the same train of thought, but thank god we have @stevejgordon :)

Comment (stevejgordon) [reply output/typescript/types.ts:7876]: Haha! Thank God I'd recently been poking around those APIs! 😄

**Pull request merged**